keps/sig-node/3960-pod-lifecycle-sleep-action/README.md

```diff
@@ -252,6 +256,16 @@ N/A
   4. For each termination grace period value, delete the pod and observe the time it takes for the container to terminate.
   5. Verify that the container is terminated after the min(sleep, grace).
 
+Tests List
+- [pod-lifecycle-sleep-action test](https://github.com/kubernetes/kubernetes/blob/a1ffdedf782edf1472102b0b99c1467d4ed39753/test/e2e/common/node/lifecycle_hook.go#L550)
+- [x]Basic functionality 
+  - Added in alpha
+- [x]Interaction with termination grace period
+  - Added in alpha
+- []Sleep duration boundary testing
```

Comment:
```suggestion
- [x]Sleep duration boundary testing
```
? Is this done in alpha? Or are you going to add it in beta?

Reply:
The last two are going to be added in beta.

Reply:
I'd make that clear in the doc.

Reply:
I added a new line under the test case name before, like this:
```
- [x]Interaction with termination grace period
  - Added in alpha
```
It may be not that clear, so let me just add `alpha/beta` after the case name
```
- [x]Interaction with termination grace period(alpha)
```
It's updated.